python/packages/ai/teams/ai/clients/openai/openai_client.py

```diff
@@ -0,0 +1,50 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from typing import List, Optional, Union
+
+from aiohttp import ClientResponseError, ClientSession
+
+from .openai_client_error import OpenAIClientError
+from .openai_client_response import OpenAIClientResponse
+from .schemas import CreateModerationResponse
+
+
+class OpenAIClient(ClientSession):
+    """
+    OpenAI Http Client
+    """
+
+    def __init__(
+        self,
+        api_key: str,
+        base_url: str = "https://api.openai.com",
+        organization: Optional[str] = None,
+    ) -> None:
+        super().__init__(
+            base_url=base_url,
+            headers={
+                "Content-Type": "application/json",
+                "User-Agent": "Microsoft Teams Conversational AI SDK",
+                "Authorization": f"Bearer {api_key}",
+            },
+        )
+
+        if organization:
+            self.headers.add("OpenAI-Organization", organization)
+
+    async def create_moderation(
+        self, input: Union[str, List[str]], model: Optional[str] = None
+    ) -> OpenAIClientResponse[CreateModerationResponse]:
+        try:
+            res = await self.post("/v1/moderations", data={"input": input, "model": model})
```

Comment:
If `model` is `None`, will OpenAI return error? Should we remove the `model` property in this case?

Reply:
if it is done they default to gpt 3.5 I bellieve, it is an optional param in the other SDK's as well. https://github.com/microsoft/teams-ai/blob/main/dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/OpenAI/OpenAIClient.cs#L39